app/javascript/src/components/Dashboard/Notes/MenuBar.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+
+import { Search, Settings, Plus } from "neetoicons";
+import { Typography } from "neetoui";
+import { MenuBar } from "neetoui/layouts";
+
+const NotesMenuBar = ({ showMenu }) => {
+  const [isSearchCollapsedSegments, setIsSearchCollapsedSegments] =
+    useState(true);
+  const [isSearchCollapsedTags, setIsSearchCollapsedTags] = useState(true);
+
+  return (
+    <div className="flex">
+      <MenuBar showMenu={showMenu} title="Notes">
+        <MenuBar.Block active count={200} label="All" />
+        <MenuBar.Block count={80} label="Users" />
+        <MenuBar.Block count={60} label="Leads" />
+        <MenuBar.Block count={60} label="Visitors" />
+        <MenuBar.SubTitle
+          iconProps={[
+            {
+              icon: Search,
+              onClick: () =>
+                setIsSearchCollapsedSegments(!isSearchCollapsedSegments),
```

Comment:
Go through the Slack review thread and fix the issue with this line of code.

Reply:
Hello Yedhin, Just to clarify.
 Are you suggesting me to do it like:
 `setIsSegmentsSearchCollapsed(isSegmentsSearchCollapsed => !isSegmentsSearchCollapsed)`
?

Reply:
@shivamkrs89 Yes correct. Please read https://academy.bigbinary.com/learn-react/pages/know-when-to-use-previous-state-for-state-update